tensorbay/exception.py

```diff
@@ -208,7 +208,7 @@ class NameConflictError(ResponseError):
 
     """
 
-    STATUS_CODE = 400
+    STATUS_CODE = 409
```

Comment:
It will be better to mention how you change the status code in the commit message.
maybe: 
`refactor(exception): change status the code of "NameConflictError" to 409`
or:
`refactor(exception): change status the code of "NameConflictError" from 400 to 409`